msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

```diff
@@ -70,7 +70,7 @@ Meetings enable collaboration, partnership, informed communication, and shared f
 <br>
 You can create scenes for meetings, provide notifications to users, populate in-meeting dialogs, and more with meeting app extensibility.
 
-Third-party and line-of-business apps built for meetings and calls are available in Government Community Cloud (GCC) but aren't available for GCC-High and Department of Defense (DOD) tenants.
+Third-party and line-of-business apps built for meetings and calls are available only in Government Community Cloud (GCC) and GCC-High environments but aren't available for Department of Defense (DOD) environment.
```

Comment:
Here we say DoD is not available, but the checkboxes below show as available. If this PR is for both the gcch and dod releases, then it should say "Third-party and line-of-business apps built for meetings and calls are available in Government Community Cloud (GCC),  GCC-High and Department of Defense (DOD),"
If the PR is for GCCH only, then the checkbox for dod should be removed below.

Reply:
disregard as this PR only applies to gcch.